advocacy_docs/pg_extensions/wait_states/installing.mdx

```diff
@@ -7,8 +7,46 @@ navTitle: Installing
 
 To install EDB Wait States on a Linux system:
 
-1.  Install EDB Wait States with the `edb-asxx-server-edb_wait_states` RPM package, where `xx` is the EDB Postgres Advanced Server version number.
+1.  Enter the following command:
 
+  ```shell
+  sudo <package-manager> -y install <install-command>
+  ```
+
+  Where: 
+  
+    - `<package-manager>`is the package manager used with your operating system:
+
+      | Package manager | Operating system     |
+      | --------------- | -------------------------------- |
+      | dnf             | RHEL 8/9 and derivatives         |
+      | yum             | RHEL 7 and derivatives, CentOS 7 |
+      | zypper          | SLES                             |
+      | apt-get         | Debian and derivatives           |
+
+    - `<install-command>` is the command required for the type of Postgres distribution you're using. In these commands `<version>` is the version of the Postgres distribution. 
+
+      | Type of Postgres distribution                                   | Install command                             |
+      | --------------------------------------------------------------- | ------------------------------------------- |
+      | PostgreSQL                                                      | postgresql`<version>`-edb_wait_states       |
```

Comment:
The same as with EPAS, for Postgres the packages differ in name, having `_` the RHEL packages and `-` the Debian ones

Reply:
Yes, that is true and not only for this package but for other packages as well , e.g 

Debian/Ubuntu
sudo apt-get -y install  edb-as16-hdfs-fdw 
                                      edb-as16-mongo-fdw 
                                      edb-as16-mysql8-fdw 
                                     edb-as15-pg-failover-slots-1

 RPM
yum -y install               edb-as16-hdfs_fdw 
                                      edb-as16-mongo_fdw 
                                     edb-as16-mysql8_fdw 
                                    edb-as15-pg-failover-slots1